versions.yaml

```diff
@@ -104,9 +104,8 @@ assets:
         tag: "TDX-v3.1"
       snp:
         description: "VMM that uses KVM and supports AMD SEV-SNP"
-        url: "https://github.com/AMDESE/qemu"
-        branch: "snp-v3"
-        commit: "ffa95097ee"
+        url: "https://github.com/Alex-Carter01/qemu-amdese-fork"
```

Comment:
Was there some issue with a build script that could not handle a commit hash instead of a tag? I'd be tempted to fix that instead of using a forked repo, though that's just my preference. I'm happy to stand corrected

Reply:
Thanks!